src/core/Akka/Actor/LocalActorRef.cs

```diff
@@ -110,7 +110,7 @@ public override IActorRefProvider Provider
             get { return _cell.SystemImpl.Provider; }
         }
 
-        /// <inheritdoc cref="IInternalActorRef"/>
```

Comment:
Shouldn't this, in theory, be correct? This is an internal CREF - it's external ones that have been failing.